src/scribe_data/cli/cli_utils.py

```diff
@@ -52,6 +62,129 @@
 }
 
 
+def get_available_languages() -> list[tuple[str, str]]:
```

Comment:
Here we'd just return the languages for the `check_language_metadata.py` file :) There's a degree of hard coding here with the `query_verbs.sparql` that we should avoid.

Maybe there is some value with letting a Scribe-Data developer set the metadata file with a language and a QID of their choice though 🤔 They could also use the QID during testing, but then that has specific return functionality now that's getting all data types by default instead of what's in Scribe-Data already. This would allow us to do a simple command in the docs to update the metadata file, but we should make this clear that it's for development only. We don't want the end user to add a language-qid pair to the metadata as they wouldn't have the ability to write the queries (if say they install with PIP).

Reply:
You are right. I thought about it only returning language names but I felt we could do better or create like a "standard". i.e. We will have to ensure that we have `query_verbs.sparql` exists for all directories. And I've checked, the `French` directory has`query_verbs_1.sparql` and `query_verbs_2sparql`, so I guess it won't be helpful.